Signal/src/ViewControllers/NewGroupViewController.m

```diff
@@ -495,7 +495,7 @@ - (void)createGroup
                       // This will save the message.
                       [message updateWithCustomMessage:NSLocalizedString(@"GROUP_CREATED", nil)];
 
-                      dispatch_async(dispatch_get_main_queue(), ^{
+                      dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0), ^{
```

Comment:
`OWSMessageSender`'s send... methods (and consequently `ThreadUtil`'s send... methods) are now thread-safe.